### ISSUE TITLE
Customizing airports and years with the groundcontrol package

### ISSUE THREAD

**Issue by jrnold** [hadley/nycflights13#22]
I wrote a package [groundcontrol](https://github.com/jrnold/groundcontrol) that adapted the code in **nycflights13** to allow the user to create a package like this one, but specify the airports, year, and whether they want to include flight to or from those airports. Would you have any interest in (a) including those functions inside this package, or (b) sharing a common codebase?

**Comment by hadley** [comment]
Sounds cool! Maybe nycflights13 could just use your code?

**Comment by jrnold** [comment]
That's what I was thinking. And a nice thing about it is that people using *R for Data Science* for pedagogy can easily customize the examples using the flights data for their own classes.

**Comment by jrnold** [comment]
Probably not at the moment.  The current version of groundcontrol is now out of date  and won't work due to changes in the formats of the source data. @jayleetx and @simonpcouch contacted me about an update to the NYC flights data https://github.com/jayleetx/nycflights and possibly essentially maintaining the equivalent of groundcontrol (though starting with the code from their own repository).
So we should resolve what the future of groundcontrol is before moving forward?  

@jayleetx @simonpcouch: What do you want to do about groundcontrol?

**Comment by simonpcouch** [comment]
The most up-to-date work on the groundcontrol-esque package is here: https://github.com/simonpcouch/anyflights . We started with `nycflights`, which @jrnold linked to previously, which implemented the year as an argument, but not the airport. Then I started on anyflights, building from the code we had written, to add in airport as an argument. As of right now, I could have the code for anyflights functional in a week or so, and ~up to CRAN standards a week or so after that. For now, should I focus on getting this running as a stand-alone package, or as a set of functions that can be easily integrated into `nycflights13`? (By the way, we submitted `nycflights` to CRAN, and they said to either further diversify the package from `nycflights13` or try to have it integrated into `nycflights13`).
